cogs/Moderation.py

```diff
@@ -15,6 +15,8 @@
 # hashlib just returns a bytes object, so this allows for slightly stricter typing
 Hash: TypeAlias = bytes
 
+MULTIPOST_EMOJI = ":multipost:1046614585760813147"
```

Comment:
## ⚠️⚠️ **WANING** ⚠️⚠️

**DO NOT MERGE** UNTIL THIS CONSTANTIS UPDATED!!!

This is currently pointing to a custom emoji on my personal testing server, since the emoji has not been uploaded to the HWH server yet.

Reply:
Pass variable to env?

Reply:
> Pass constant to env?

I think we can leave it as is